### DISCUSSION TITLE
Outdoor light with PIR or Sonar

### DISCUSSION THREAD

**Discussion by elajoie** [arendst/Tasmota#10878] [Q&A] [answered]
Hi all,

Has anyone seen a good option for a outdoor light with either PIR or Sonar which we can run tasmota on?

Thanks,
Eric

**Comment by barbudor** [comment, marked as answer]
I would pickup a standard cheap one and tasmotize it with an ESP

**Reply by elajoie** [reply]
Agree, that's my backup plan. I looked at the sonar option and it looks like I would need a 5V driver which is not possible with the Sonoff options and for the PIR it looks like an easy design but I was hoping some manufacturer already had an 8266 or ESP32 based model.

**Reply by sfromis** [reply]
Such commercial lights with auto-on are typically without wifi, and then they'd not use an ESP8266 or ESP32. "Brain surgery" may be quite doable still. As many PIR sensor boards have adjustable timing anyway, they'd not even need any MCU in there.

**Reply by barbudor** [reply]
I'm pretty sure I've seen some discussions on Discord in December or January about such light. But can't remember who/when
You may have more luck asking there
